### ISSUE TITLE
Show the average purchase price in the legend of the chart

### ISSUE THREAD

**Issue by gulfwarden** [buchen/portfolio#3014]
**Is your feature request related to a problem? Please describe.**
I often look at the chart of stocks I already bought in the past. At the moment, it is from my perspective very hard to see the average purchase price in a easy way.

**Describe the solution you'd like**
I would be great, if the average purchase price could be added to the legend (like it is done with the limits in this example).

![image](https://user-images.githubusercontent.com/37246732/197721644-e457cf05-8b32-4303-9578-3358a5908e1a.png)

**Comment by ma4nn** [comment]
@gulfwarden not sure if I did understand your request correctly. You want to add the value of the average purchase price to the legend of the security chart, e.g. like this: "Einstandspreis (gleitender Durchschnitt): XX,XX €"?

But the average purchase price is not a constant value over time, so which value should be shown there and what is the benefit?

**Comment by gulfwarden** [comment]
Yes, that´s correct, like this: "Einstandspreis (gleitender Durchschnitt): XX,XX €

The advantage is, that i don´t need to switch to "Vermögensaufstellung" to see the average purchase price over all purchases and depots.

**Comment by Sn1kk3r5** [comment]
In the meanwhile available in the charts
